### DISCUSSION TITLE
Context menu in GitHub?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#31497] [Other Feature Feedback, Questions, & Ideas] [answered]
I would like to have a context menu so that I can easily rename .txt .md etc files without having to go inside folders to make commits. Software like Ghost Writer, Adobe Photoshop, Gimp have a context menu to facilitate certain actions with the mouse.

#### idea/concept
![image](https://user-images.githubusercontent.com/111701513/187797727-96e6c362-0116-4e11-8d68-4a10817d6626.png)

**Comment by dikkyryan** [comment, marked as answer]
check https://github.com/orgs/community/discussions/31486
